docker-compose.yml

```
@@ -34,6 +36,9 @@ services:
       sh -c "
         mongod --replSet rs0 --bind_ip_all --logpath /var/log/mongodb.log --logappend &
         sleep 5 &&
-        mongosh --eval 'try { rs.initiate() } catch(e) { print(e) }' &&
+        mongosh --eval '
+          var config = { _id: \"rs0\", members: [ { _id: 0, host: \"$MONGO_REPLICA_HOST\" } ] };
+          try { rs.initiate(config); } 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)